src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -63,6 +68,86 @@ public void sendEmail(CaseDetails caseDetails) {
 
     }
 
+    public void sendEmailToApplicantAndRespondent(CaseDetails caseDetails) {
+        log.info("Sending the new manage order emails for Applicant/Respondent {}",caseDetails.getId());
+        CaseData caseData = emailService.getCaseData(caseDetails);
+        SelectTypeOfOrderEnum isFinalOrder = caseData.getSelectTypeOfOrder();
+        if (caseData.getCaseTypeOfApplication().equalsIgnoreCase("C100")) {
+            Map<String, String> applicantsMap = getEmailPartyWithName(caseData
+                                                                         .getApplicants());
+            Map<String, String> respondentMap = getEmailPartyWithName(caseData
+                                                                         .getRespondents());
+            for (Map.Entry<String, String> appValues : applicantsMap.entrySet()) {
+
+                emailService.send(
+                    appValues.getKey(),
+                    (isFinalOrder == SelectTypeOfOrderEnum.finl) ? EmailTemplateNames.CA_DA_FINAL_ORDER_EMAIL
+                        : EmailTemplateNames.CA_DA_MANAGE_ORDER_EMAIL,
+                    buildApplicantRespondentEmail(caseDetails, appValues.getValue()),
+                    LanguagePreference.english
+                );
```

Comment:
These lines of code is exactly duplicate of code from line# 93-99. Can we move this to a single method to reuse it at multiple places?

Reply:
created common method to address this